packages/connect-extension-protocol/fix-package-type

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+#   Add package.json files to cjs/mjs subtrees
+#
+
+cat >dist/cjs/package.json <<!EOF
+{
+    "type": "commonjs"
+}
+!EOF
+
+cat >dist/mjs/package.json <<!EOF
+{
+    "type": "module"
+}
+!EOF
```

Comment:
Why is this needed? There's already a `package.json`, why do we need two additional ones?

Reply:
Its not exactly "two additional ones", rather "two  complementary ones". These `package.json` files define the default package type for each of the `dist/*` sub-directories. Main `package.json` does not have a type property (if it did - hybrid solution would not work), and with this script it is pushed down to the `package.json` files under `dist/`.

Reply:
It doesn't answer my question: _why_ do we need them?

Reply:
When `type` is set to `module` it clarifies that this is an ESM package - we could avoid that by changing the `js` extension to `mjs` but I find it unnecessary as a simple `"type" : "module"` would do the same.